themes/dracula.omp.json

```diff
@@ -58,8 +58,24 @@
         }
       ],
       "type": "prompt"
+    },
+    {
+      "alignment": "left",
+      "segments": [
+        {
+          "background": "#f1fa8c",
+          "foreground": "#282a36",
+          "invert_powerline": true,
+          "leading_diamond": "\ue0b2",
+          "style": "diamond",
+          "template": " \ue7ad {{.Profile}}{{if .Region}}@{{.Region}}{{end}}",
+          "trailing_diamond": "\ue0b4",
+          "type": "aws"
+        }
+      ],
+      "type": "rprompt"
     }
   ],
   "final_space": true,
-  "version": 2
+  "version": 3
```

Comment:
The version number needs to remain the same. It's the config schema version, not the version of the config as it changes.